plugins/woocommerce-blocks/tests/e2e/tests/product-filters/product-filters.block_theme.spec.ts

```diff
@@ -507,4 +507,38 @@ test.describe( `${ blockData.name }`, () => {
 			await expect( productFilterAttributeBlockHeading ).toBeVisible();
 		} );
 	} );
+
+	test( 'should be able to be added to the Product Filters Parent wrapper block', async ( {
+		editor,
+		pageObject,
+	} ) => {
+		await pageObject.addProductFiltersBlock( { cleanContent: true } );
+
+		const block = editor.canvas.getByLabel(
+			'Block: Product Filters (Experimental)'
+		);
+		await expect( block ).toBeVisible();
+
+		const searchTerms = [
+			'status (experimental)',
+			'price (experimental)',
+			'rating (experimental)',
+			'attribute (experimental)',
+			'active (experimental)',
+		];
+
+		const addBlockButton = block.getByRole( 'button', {
+			name: 'Add block',
+		} );
+		await addBlockButton.dispatchEvent( 'click' );
```

Comment:
Here you can use the `click` method instead:

```suggestion
		await addBlockButton.click();
```